recipes/default.rb

```diff
@@ -246,7 +251,10 @@
          :public_ip => public_ip,
          :dela_ip => dela_ip,
          :krb_ldap_auth => node['ldap']['enabled'].to_s == "true" || node['kerberos']['enabled'].to_s == "true",
-         :hops_version => get_hops_version(node['hops']['version'])
+         :hops_version => get_hops_version(node['hops']['version']),
+         :onlinefs_password => encrypted_onlinfefs_password,
+         :onlinefs_email => node['onlinefs']['hopsworks']['password'],
```

Comment:
2 points here:
- there is a typo here, it should be email not password
- If you are using `node` without any additional logic, you don't need to pass them as variables, you can just use the node object in the template. See below.

Reply:
1. good catch, thanks
2. didn't know, will replace

Reply:
1. good catch, thanks
2. didn't know, will replace